golangci.yaml

```diff
@@ -1,12 +1,35 @@
-linters:
-  disable:
-    - megacheck
+liners:
+  disable-all: true
   enable:
+    - bodyclose
+    - deadcode
+    - depguard
+    - dogsled
+    - errcheck
+    - gocyclo
+    - gofmt
+    - goimports
+    - goprintffuncname
+    - gosec
     - gosimple
+    - govet
+    - ineffassign
+    - interfacer
+    - nakedret
+    - nolintlint
+    - rowserrcheck
+    - scopelint
     - staticcheck
+    - structcheck
+    - typecheck
+    - unconvert
+    - varcheck
+    - gocritic
     - unused
-    - interfacer
-    - errcheck
-    - prealloc
-    - gosec
+    - misspell
+    - dupl
```

Comment:
`dupl` is nice but may be a source of headaches when using `client-go` structs or api-machinery's `metav1` package.

Reply:
Moved it to the wish list as it is reporting some very valid cut/paste abuse in the tests and I'd rather not take on any more in this merge. Would like to enable later.